conf/layer.conf

```diff
@@ -8,4 +8,4 @@ BBFILE_COLLECTIONS += "meta-rpi"
 BBFILE_PATTERN_meta-rpi := "^${LAYERDIR}/"
 BBFILE_PRIORITY_meta-rpi = "16"
 
-LAYERSERIES_COMPAT_meta-rpi = "thud"
+LAYERSERIES_COMPAT_meta-rpi = "dunfell gatesgarth"
```

Comment:
I presume this actually gives us RPI4B support as well, since those are new enough to support? (NOTE! Not going to CLAIM that anywhere publicly, since we have not tried it - but I think in theory at least).

Reply:
Yes, Dunfell should have that capability. But yeah you are correct, it hasn't been verified to be functional